### ISSUE TITLE
CFT failover, why no command 003- to restart restjavad/restnoded?

### ISSUE THREAD

**Issue by amolari** [F5Networks/f5-aws-cloudformation#137]
## Do you already have an issue opened with F5 support?
No
## Description
In CFT Failover v5.8.0 (and previous 5.7.1) you have configured
```
"commands": {
       "000-disable-1nicautoconfig": {
        "command": "/usr/bin/setdb provision.1nicautoconfig disable"
       },
       "001-rest-provision-extramb": {
        "command": "/usr/bin/setdb provision.extramb 500"
       },
       "002-rest-use-extramb": {
        "command": "/usr/bin/setdb restjavad.useextramb true"
       },
       "010-install-libs": {
```

Unlike the CFT autoscale, there is no
```
"003-rest-restart": {
        "command": "bigstart restart restjavad restnoded"
       },
```

any reason for that?`

## Template
CFT supported/failover/across-net/via-api/2nic/existing-stack/bigiq/    v5.7.1 / 5.8.0

## Severity Level
Severity: 5

**Comment by shyawnkarim** [comment]
Thanks for reporting this inconsistency. I'm not sure why the autoscale templates restart restjavad and restnoded and the failover ones do not but it appears as if it isn't necessary to do so. I've created a bug ticket for this, internal ID ESECLDTPLT-2350, to look into it.

**Comment by shyawnkarim** [comment]
Closing due to age. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-aws-cloudformation-v2).
